runners/flink/job-server/build.gradle

```diff
@@ -31,6 +31,7 @@ mainClassName = "org.apache.beam.runners.flink.FlinkJobServerDriver"
 
 dependencies {
   compile project(path: ":beam-runners-flink_2.11", configuration: "shadow")
+  compile project(path: ":beam-sdks-java-extensions-google-cloud-platform-core", configuration: "shadow")
```

Comment:
Do we have to add a hard dependency here? If the idea is to make the distributed filesystem available for all users, we might as well include _all_ supported beam FS backends.

@jkff What do you think?

Reply:
Agreed. We do need a hard dependency - we're building a self-contained jar. The others would be just AWS and HDFS I suppose.

Reply:
Yes, I think all the extension should be added to job server uber jar.